_docs/tasks/security/role-based-access-control.md

```diff
@@ -0,0 +1,355 @@
+---
+title: Setting up Istio Role-Based Access Control
+overview: This task shows how to set up role-based access control for services in Istio mesh.
+
+order: 30
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+This task shows how to set up role-based access control (RBAC) for services in Istio mesh. You can read more about Istio
+RBAC from [Istio RBAC concept page](https://istio.io/docs/concepts/security/rbac.html).
+
+## Before you begin
+
+* Set up Istio on auth-enabled Kubernetes by following the instructions in the
+  [quick start]({{home}}/docs/setup/kubernetes/quick-start.html).
+  Note that authentication should be enabled at step 5 in the
+  [installation steps]({{home}}/docs/setup/kubernetes/quick-start.html#installation-steps).
+
+* Deploy the [BookInfo]({{home}}/docs/guides/bookinfo.html) sample application.
+
+ > Note: Some sample configurations we use below are not in the current Istio release yet. So before you continue, you
+ need to copy the following configuration files from https://github.com/istio/istio/tree/master/samples/bookinfo/kube to
+ "samples/bookinfo/kube" directory under where you installed Istio. The files include `bookinfo-add-serviceaccount.yaml`
+ (replace the original one), `istio-rbac-enable.yaml`, `istio-rbac-namespace.yaml`, `istio-rbac-productpage.yaml`,
+ `istio-rbac-details-reviews.yaml`, `istio-rbac-ratings.yaml`.
+
+* Run the following command to
```

Comment:
It might help to explain why we need to do this. Something like
"In this example, we will enable access control based on Service Accounts, which are cryptographically authenticated in the Istio mesh. In order to give different microservices different access privileges, we will create some service accounts and redeploy BookInfo microservices running under them."

Reply:
Spike, thanks for the review! That's a good point. I have added the explanation.